client/src/components/Board/mockData.tsx

```diff
@@ -0,0 +1,38 @@
+import { IBoardData } from '../../interface/Board';
+
+const mockData: IBoardData = {
+  tasks: {
+    'task-1': { id: 'task-1', content: 'Essay on the environment', tag: 'green' },
+    'task-2': { id: 'task-2', content: 'Midterm exam', date: 'March 10', tag: 'red' },
```

Comment:
I think we should probably add support for our dates to be Date objects and not strings but otherwise, this is good work!